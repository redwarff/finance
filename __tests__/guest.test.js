import { shallow } from 'enzyme'
import React from 'react'

import Guest from 'components/Guest.jsx'

describe("Guest", function() {
  const guest = { guest: 'Mariah' }

  it("renders guest's name", () => {
    const wrapper = shallow(<Guest guest={guest} />)

    expect(wrapper.find('div').at(0).text()).toBe('Mariah')
  })

  it('renders correctly (snapshot)', () => {
  const wrapper = shallow(<Guest guest={guest} />)

  expect(wrapper).toMatchSnapshot();
});

})