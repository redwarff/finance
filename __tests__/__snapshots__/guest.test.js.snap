// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Guest renders correctly (snapshot) 1`] = `
<div>
  Mariah
</div>
`;
